docs/HowTo/Extending-EthSigner/Add-Signer.md

```diff
@@ -4,26 +4,26 @@ description: Adding an external signer to EthSigner
 
 # Creating an external signer for EthSigner
 
-EthSigner supports the implementation of additional external signers in same way as Hashicorp Vault.
+EthSigner supports the implementation of additional external signers in the same way as Hashicorp Vault.
 
 To create an external signer:
 
 1. Clone the [EthSigner repository](https://github.com/PegaSysEng/ethsigner).
 
-1. Create a gradle module for the new signer in the `ethsigner/signer` directory.
+1. Create a Gradle module for the new signer in the `ethsigner/signer` directory.
 
-1. To ensure the module for the new signer is built, add the module to the `settings.gradle` file.
+1. To include the module in the build, add the module to the `settings.gradle` file.
 
 1. In the new module, create a concrete child class of `SignerSubCommand`.
 
     The new `SignerSubCommand` must include the parameters (tagged with PicoCLI annotations) to
     initialize your signer.
 
-1. Update the `createSigner()` function to be the entry point to create and return your signer.
+1. Make the `createSigner()` function the entry point to create and return your signer.
```

Comment:
Make the `createSignerFactory()` function  the entry point to create and return your TransactionSignerProvider (a class capable of supplying 1 or more TransactionSigners (which in turn is able to sign transactions using the mechanism in development)).

Reply:
Your newly created TransactionSupplierProvider should be exposed only as such (i.e. as the base interface).

Reply:
I'm unsure about using "should be" here. Are we using it to express advice or an expectation?

Reply:
Reverted to original - comments covered by following sentence and note.